roles/edpm_neutron_dhcp/tasks/install.yml

```diff
@@ -19,8 +19,8 @@
     path: "{{ item.path }}"
     setype: "container_file_t"
     state: directory
-    owner: "{{ item.owner | default(lookup('pipe', 'whoami')) }}"
-    group: "{{ item.group | default(lookup('pipe', 'whoami')) }}"
+    owner: "{{ item.owner | default(ansible_user_id) }}"
```

Comment:
I think would be good to do | default(ansible_user) | default(ansible_user_id) instead as suggested in Jira ticket. As ansible_user_id relies on gathered facts and that can change based on become: yes used or not. Molecule test failed as that using local connection and ansible_user is not defined in that case. Having the above should work for molecule too unless and until ansible_user explicitly set there.